src/main/java/org/demesup/AppController.java

```diff
@@ -0,0 +1,69 @@
+package org.demesup;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+import org.demesup.controller.Controller;
+import org.demesup.model.Model;
+import org.hibernate.ObjectNotFoundException;
+import org.utils.Read;
+import org.utils.exception.ExitException;
+
+import java.io.IOException;
+import java.util.Map;
+import java.util.function.Consumer;
+import java.util.function.Function;
+
+import static org.utils.Read.readEnumValue;
+
+@Slf4j
+public class AppController {
+    @Getter
+    @AllArgsConstructor
+    public
```

Comment:
internal classes are better to locate at the bottom of the class.  But the best is separate file